dqc_us_rules/dqc_us_0005.py

```diff
@@ -44,36 +62,71 @@ def _get_end_of_period(val):
 
 
 def validate_facts(val):
+    """
+    Validates facts, in other words it checks to see if the facts contained in
+    val are correctly implemented.
+
+    :param val: val to check
+    :type val: val
+    """
     eop_results = _get_end_of_period(val)
-    fact_dict = facts.LegalEntityAxis_facts_by_member(filter(lambda f: f.context is not None, val.modelXbrl.facts))
+    fact_dict = facts.legal_entity_axis_facts_by_member(
+        filter(lambda f: f.context is not None, val.modelXbrl.facts)
+    )
     for lea_member, fact_list in fact_dict.items():
-        lookup = lea_member if lea_member in eop_results else facts.LEGALENTITYAXIS_DEFAULT
+        lookup = lea_member \
```

Comment:
instead of using the \ to break up the line, should put parens around the line and break it up like normal